tools/test-proxy/Azure.Sdk.Tools.TestProxy/RecordingHandler.cs

```diff
@@ -656,12 +656,24 @@ public HttpClientHandler GetTransport(bool allowAutoRedirect, TransportCustomiza
 
                 clientHandler.ServerCertificateCustomValidationCallback = (HttpRequestMessage httpRequestMessage, X509Certificate2 cert, X509Chain x509Chain, SslPolicyErrors sslPolicyErrors) =>
                 {
-                    bool isChainValid = certificateChain.Build(cert);
-                    if (!isChainValid) return false;
-                    var isCertSignedByTheTlsCert = certificateChain.ChainElements.Cast<X509ChainElement>()
-                        .Any(x => x.Certificate.Thumbprint == ledgerCert.Thumbprint);
+                    if (!string.IsNullOrWhiteSpace(customizations.TSLValidationCertHost) && httpRequestMessage.RequestUri.Host != customizations.TSLValidationCertHost)
+                    {
+                        if (sslPolicyErrors == SslPolicyErrors.None)
```

Comment:
This is the basic implementation of the default `ServerValidationCallback`. Unfortunately the default implementation isn't stored as a Func that I can just re-use 👍